ev3dev2/wheel.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+
+"""
+Wheel and Rim classes
+
+A great reference when adding new wheels is http://wheels.sariel.pl/
+"""
+from math import pi
+
+
+class Wheel(object):
+    """
+    A base class for various types of wheels, tires, etc
+    All units are in mm
+    """
+
+    def __init__(self, diameter_mm, width_mm):
+        self.diameter_mm = float(diameter_mm)
+        self.width_mm = float(width_mm)
+        self.radius_mm = float(diameter_mm / 2)
```

Comment:
Why do we need both radius and diameter?

Reply:
Not sure what it hurts?

Reply:
One of those could be a property, so we would not need to duplicate information and keep the fields in sync.